src/dataset-builder/ConceptSearch.test.ts

```diff
@@ -135,7 +135,7 @@ describe('ConceptSearch', () => {
     const concept = displayedConcepts[0];
     // Act
     const user = userEvent.setup();
-    await user.click(await screen.findByLabelText(`add ${concept.id}`));
```

Comment:
should we still use add/remove or does `check` and `uncheck` work?

Reply:
I like `check`/`uncheck`.

Reply:
Although, actually, we don't need `aria-label` here, since if you look at the `LabeledCheckbox` component, it includes an `aria-labelledby` property. If you move the `HighlightConceptName` text to be a child of the `LabeledCheckbox` component, then `aria-labelledby` and `aria-checked` perform the more correct approach to checking this.

Then, you could also inline the checked variable, and not use an intermediary variable.